plugins/vpp/l2plugin/bd_config_test.go

```diff
@@ -0,0 +1,551 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package l2plugin_test
```

Comment:
please add following scenarios:

1)
- create a bridge domain with at least 4 interfaces 
- modify bridge domain where 2 interfaces are removed and 2 new are added (the rest should remain untouched)
- BVI move from one of the removed interfaces to one of the added
- verify that required interfaces are configured (check the metadata)
It verifies that the diff calculation of interfaces and BVI resolution is correct while interface count in every case is > 1

2)
- simple error case test where bridge domain with 2 BVI interfaces is configured

Reply:
Added both